tests/test-quantize-perf.cpp

```diff
@@ -36,8 +37,10 @@ struct quantize_perf_params {
     bool op_dequantize_row_q = false;
     bool op_quantize_row_q_dot = false;
     bool op_vec_dot_q = false;
+    int64_t iterations = ITERATIONS;
 };
 
+int64_t iterations = ITERATIONS;
```

Comment:
Avoid the global state - pass it as function argument

Reply:
I revised code as iterations parameter send through function argument.